src/main/java/frc/robot/RobotMap.java

```diff
@@ -27,20 +27,21 @@ private RobotMap() {}
     // coral elevator
     public static final int CORAL_ELEVATOR_UPPER_LIMIT_SWITCH = 2;
     public static final int CORAL_ELEVATOR_LOWER_LIMIT_SWITCH = 3;
-    public static final int CORAL_ELEVATOR_PISTON1_FORWARD_CHANNEL = 0;
-    public static final int CORAL_ELEVATOR_PISTON1_REVERSE_CHANNEL = 1;
-    public static final int CORAL_ELEVATOR_PISTON2_FORWARD_CHANNEL = 2;
-    public static final int CORAL_ELEVATOR_PISTON2_REVERSE_CHANNEL = 3;
-
+    public static final int CORAL_ELEVATOR_PISTON1_FORWARD_CHANNEL = 13;
+    public static final int CORAL_ELEVATOR_PISTON1_REVERSE_CHANNEL = 15;
+    public static final int CORAL_ELEVATOR_PISTON2_FORWARD_CHANNEL = 13;
+    public static final int CORAL_ELEVATOR_PISTON2_REVERSE_CHANNEL = 15;
     // algae gripper
-    public static final int ALGAE_GRIPPER_MOTOR = 4;
+    public static final int ALGAE_GRIPPER_MOTOR1 = 7;
+    public static final int ALGAE_GRIPPER_MOTOR2 = 6;
+
     public static final int ALGAE_GRIPPER_DIGITALINPUT = 4;
 
     // algae arm
     public static final int ALGAE_ARM_SWITCH_TOP = 5;
     public static final int ALGAE_ARM_SWITCH_BOTTOM = 6;
-    public static final int ALGEA_ARM_FORWARD_PISTON_FORWARD_CHANNEL = 4;
-    public static final int ALGEA_ARM_FORWARD_PISTON_REVERSE_CHANNEL = 5;
+    public static final int ALGEA_ARM_FORWARD_PISTON_FORWARD_CHANNEL = 12;
+    public static final int ALGEA_ARM_FORWARD_PISTON_REVERSE_CHANNEL = 14;
     public static final int ALGEA_ARM_BACKWARD_PISTON_FORWARD_CHANNEL = 6;
```

Comment:
remove constants for solenoids which are not directly connected to the PH. Same as you removed the code for the solenoids, they should not exist in code anymore